compose.yml

```diff
@@ -6,30 +6,36 @@ services:
       - postgres_data:/var/lib/postgresql/data
       - ./db/postgres:/docker-entrypoint-initdb.d
     environment:
-      POSTGRES_HOST_AUTH_METHOD: trust
+      POSTGRES_USER: ats
+      POSTGRES_PASSWORD: password
+      POSTGRES_DB: ats_docker_production
     ports:
       - "5432:5432"
 
   web:
     build:
       context: .
-      dockerfile: development.Dockerfile
+      dockerfile: Dockerfile
       args:
-        - NODE_ENV=development
+        NODE_ENV: production
```

Comment:
Why do we need a variable for node in production?
I assume that this variable is only needed for installing packages and compiling assets, and they occur before the service is started

Reply:
will remove, but Dockerfile still uses this variable (default value)